app/src/main/java/com/amaze/filemanager/ui/notifications/FtpNotification.java

```diff
@@ -49,46 +49,31 @@ private void createNotification(Context context, boolean noStopButton) {
                 + address.getHostAddress() + ":"
                 + port + "/";
 
+        NotificationCompat.Builder builder = new NotificationCompat.Builder(context.getApplicationContext(), NotificationConstants.CHANNEL_FTP_ID);
```

Comment:
You rebuild the entire thing here. You should try to centralize building to a single method.

Reply:
It looks like this is the only item that isn't resolved. Does it still need to be done? I can probably do it Friday or Saturday.

Reply:
I left it to you because I don't want to break everything :smile:.

Reply:
Haha! Thanks for leaving me one thing to break. 😄